instance/conn_name.go

```diff
@@ -74,3 +75,9 @@ func ParseConnName(cn string) (ConnName, error) {
 	}
 	return c, nil
 }
+
+// ConnectionNameResolver resolves  This allows an application to replace the default
```

Comment:
Let's clean up this to make is a complete sentence starting with the interface name.

Reply:
Fixed.